### PR TITLE
virtual_disks_gluster: start ga by prepare_guest_agent

### PR DESCRIPTION
**Pull request by PandaWei**
replace check_vm_guestagent() with prepare_guest_agent.

### CODE REVIEW

File: libvirt/tests/src/virtual_disks/virtual_disks_gluster.py [hunk 1]
```diff
@@ -196,11 +168,10 @@ def test_pmsuspend(vm_name):
                 vmxml_backup.define()
                 raise error.TestNAError("Cann't create the domain")
 
-        session = vm.wait_for_login()
         # Run the tests.
         if pm_enabled:
             # Makesure the guest agent is started
-            check_vm_guestagent(session)
+            vm.prepare_guest_agent()
```
Comment (yangdongsheng) [inline libvirt/tests/src/virtual_disks/virtual_disks_gluster.py]: Do we need a try here?

Comment (PandaWei) [inline libvirt/tests/src/virtual_disks/virtual_disks_gluster.py]: Thanks for your reminder.
updated it.

File: libvirt/tests/src/virtual_disks/virtual_disks_gluster.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 from virttest import virsh
 from virttest import data_dir
 from virttest import utils_misc
+from virttest import aexpect, virt_vm, remote
```
Comment (waynesun09) [inline libvirt/tests/src/virtual_disks/virtual_disks_gluster.py]: aexpect is not used

**Comment by PandaWei** [comment]
@waynesun09 
Thanks for your kind of reminder.

update it

**Comment by waynesun09** [comment]
LGTM, got an error on:

```
Test gluster.disk_qcow2.test_dompmsuspend has encountered error
  INFO | ERROR type_specific.io-github-autotest-libvirt.virtual_disks.gluster.disk_qcow2.test_dompmsuspend -> CmdError: Command <gluster volume info gluster-vol1> failed, rc=1, Command returned non-zero exit status    [context: (gluster_vol_stop) --> Gluster volume info failed for volume: gluster-vol1]
```

it's not caused by update here, I'll invite @rbian to take a look.

**Comment by rbian** [comment]
@waynesun09 , patch is fine. 
I think the error may be casued by your environment. Any detailed log?

**Comment by waynesun09** [comment]
@rbian @PandaWei The fix here is good enough and for the failure is not caused by this PR and we will did more dig, so Merge this now, Thanks!

**Pull request merged**